### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

No the binary has stabilised we should update the README.

### Solution

* Update readme.
* Add linting for `outputs.tf`.

### Notes

- **Remember** to tag your merge commit after merging 👍
- **Remember** to add the Clubhouse ticket url as a comment ♣️ 🏠

### CODE REVIEW

**Comment by stevenjack** [comment]
https://app.clubhouse.io/vidsy/story/6762

File: terraform/linter/terraform_type.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	terraformTypeModule    terraformType = "module"
+	terraformTypeOutput    terraformType = "output"
+	terraformTypeProvider  terraformType = "provider"
+	terraformTypeResource  terraformType = "resource"
+	terraformTypeTerraform terraformType = "terraform"
+	terraformTypeVariable  terraformType = "variable"
+)
+
+type (
+	// TerraformTypes the different types or terraform resources.
```
Comment (pocockn) [inline terraform/linter/terraform_type.go]: Do we still need this comment?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+If this file exists, the following is checked:
+
+1. Should only contain 1 or more `provider` configs and one `terraform` config.
+1. Should contain no `data`, `local`, `module`, `output` or`resource` resources.
+
+### resources.tf
+
+If this file exists, the following is checked:
+
+1. Should contain 1 or more `local, `module` or `resource` resources.
```
Comment (tx3stn) [inline README.md]:
```suggestion
1. Should contain 1 or more `local`, `module` or `resource` resources.
```

**Review by pocockn — APPROVED** [review]
🍰

**Pull request merged**